libs/gi/ui/src/components/DocumentDisplay.tsx

```diff
@@ -110,6 +117,8 @@ function FieldsSectionDisplay({
   hideHeader?: boolean | ((section: DocumentSection) => boolean)
   bgt?: CardBackgroundColor
 }) {
+  const { data } = useContext(DataContext)
```

Comment:
What is this for?

Reply:
For displaying in Archive page (which doesnt have data)